Issue-Tracking.md

```diff
@@ -49,7 +49,7 @@ When team members files a bug they perform steps of the inbox tracker for the is
 
 Issues need to be verified.
 
-Verification is a service that you request from others either implicitly with the `bug`-label or explicitly with the `verification-needed`-label. Find issue that are to be verified with these queries
+Verification is a service that you request from others either implicitly with the `bug`-label or explicitly with the `verification-needed`-label. Find issues that are to be verified with these queries:
```

Comment:
I agree with adding a colon in this case, but for the sake of consistency across team documentation, we do not add colons here.